Index.html

```diff
@@ -31,7 +31,170 @@ <h3 class="connect">LET’S CONNECT</h3>
+          <ul class="softwares">
+            <li class="software">html</li>
+            <li class="software">css</li>
+            <li class="software">javascript</li>
+          </ul>
+          <button class="btn">see project</button>
+        </div>
+
+      </div>
 
+      <div class="proj3">
+        <div class="proj-content-space">
+          <h1 class="proj-title">Tonic</h1>
+          <nav class="proj-img2">
+            <h4 class="cpny">Canopy</h4>
+            <img src="./images/Counter.png" alt="counter" />
+            <span class="project-lang">Back End Dev</span>
+            <img src="./images/Counter.png" alt="counter" />
+            <span>2015</span>
+          </nav>
+          <p class="proj-desc">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="softwares">
+            <li class="software">html</li>
+            <li class="software">css</li>
+            <li class="software">javascript</li>
+          </ul>
+          <button class="btn">see project</button>
+        </div>
+
+      </div>
+
+      <div class="proj4">
+        <div class="proj-content-space">
+          <h1 class="proj-title">Multi-Post Stories</h1>
+          <nav class="proj-img2">
+            <h4 class="cpny">Canopy</h4>
+            <img src="./images/Counter.png" alt="counter" />
+            <span class="project-lang">Back End Dev</span>
+            <img src="./images/Counter.png" alt="counter" />
+            <span>2015</span>
+          </nav>
+          <p class="proj-desc">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="softwares">
+            <li class="software">html</li>
+            <li class="software">css</li>
+            <li class="software">javascript</li>
+          </ul>
+          <button class="btn">see project</button>
+        </div>
+
+      </div>
+    </div>
+
+   </section> 
+   
+   <section>
+    <div class="myself">
+      <h2 class="name">About<br/>Myself</h2>
+      <p class="text">
+        Hello I’m a software developer! I can help you build a product ,
+        feature or website Look through some of my work and experience! If you
+        like what you see and have a project you need coded, don’t hestiate to
+        contact me.
+      </p>
+      <h3 class="connect text">LET’S CONNECT</h3>
+        <ul class="socialmedia">
+          <li><a href="#"><img class="twitter" src="images\twitter.png" alt="twitter"></a></li>
+          <li><a href="#"><img class="i" src="images\linkedin.png" alt="i"></a></li>
+          <li><a href="#"><img class="m" src="images\M.png" alt="m"></a></li>
+          <li><a href="#"><img class="g" src="images\g.png" alt="g"></a></li>
+          <li><a href="#"><img class="v" src="images\v.png" alt="v"></a></li>
+        </ul>
+        <button class="resume">Get my resume</button>
```

Comment:
Kindly update the styling of your about me section to better match the figma design🙏.
Your design:
<img width="234" alt="image" src="https://user-images.githubusercontent.com/37116322/219808205-65d842d4-dae9-4c03-bc43-3f4ea23f8482.png">

Figma design:
<img width="228" alt="image" src="https://user-images.githubusercontent.com/37116322/219808404-f9824f4f-7bb7-40b2-bedf-1b017edc2d06.png">

A few things to note:
- The main title should be bolder(font-weight)
- The background color of this section should be white(#fff)
- The contents of this section should be left aligned

**Please try your best to make sure that your design matches the Figma design as much as possible😎**